eppo-sdk-test/EppoClientTest.cs

```diff
@@ -114,7 +114,7 @@ public void ShouldPollForConfigInServerMode()
 
         _client = EppoClient.Init(config);
 
-        Thread.Sleep(30);
+        Thread.Sleep(40);
```

Comment:
- Initial call (t=0)
- First poll (t=25ms)
- Wait ends (t=40ms) - Should see exactly 2 calls
- Second poll (t=50ms) - Happens after verification